docs/known_generator_issues.md

```diff
@@ -0,0 +1,82 @@
+# Known issues
+
+## Unsupported `netcoreapp3.1` TFM
+
+Tracking [issue](https://github.com/dotnet/source-build/issues/3251)
+
+Reference packages for TFM `netcoreapp3.1` are not supported. You may get numerous compilation issues like:
+
+```text
+*/lib/netcoreapp3.1/*: error CS0246: The type or namespace name 'DebuggableAttribute' could not be found
+*/lib/netcoreapp3.1/*: error CS0518: Predefined type 'System.String' is not defined or imported
+```
+
+Workaround: update the `*.csproj` file:
+
+* remove the `netcoreapp3.1` TFM from the `<TargetFrameworks>` list.
+* remove `<PropertyGroup>` and `<ItemGroup>` with the `netcoreapp3.1` condition.
+* remove the `lib\netcoreapp3.1` folder.
+
+## CLSCompliant assembly attributes
+
+Tracking [issue](https://github.com/dotnet/source-build/issues/3252)
+
+By default, it is set to true and it may produce compilation errors like:
+
+```text
+error CS3015: ... has no accessible constructors which use only CLS-compliant types
+error CS3016: Arrays as attribute arguments is not CLS-compliant
+```
+
+Workaround: update the assembly attribute `[assembly: CLSCompliant(true)]` and set value to `false`
```

Comment:
This is already fixed with the new SBRP tooling + new GenAPI. Please remove the section.

Reply:
Prepared a PR: https://github.com/dotnet/source-build-reference-packages/pull/564